cypress/e2e/tests/upgrade/create_upgrade_data.test.ts

```diff
@@ -176,4 +194,8 @@ describe(["@pre-upgrade"], "Creating pre-requisites before an upgrade", () => {
         assessmentApplication.perform_assessment("low", [stakeHolder]);
         assessmentApplication.verifyStatus("assessment", "Completed");
     });
+
+    it("Assess archetype", function () {
+        archetype.perform_assessment("low", [stakeHolder], [stakeHolderGroup]);
```

Comment:
I have one minor comment, otherwise the PR LGTM .

Add this statement to verify that archetype assessment is complete :
archetype.validateAssessmentField("Low");